src/openforms/js/components/admin/form_design/form-creation-form.js

```diff
@@ -182,6 +183,14 @@ function reducer(draft, action) {
                     draft.literals[fieldName].value = value;
                     break;
                 }
+                case 'autoLoginAuthenticationBackend': {
+                    if (draft.autoLoginAuthenticationBackend === value) {
+                        draft.autoLoginAuthenticationBackend = "";
+                    } else {
+                        draft.autoLoginAuthenticationBackend = value;
+                    }
+                    break;
+                }
```

Comment:
as mentioned before, this should not be needed if you use the proper `name` attributes/props in your form elements.